cobalt/BUILD.gn

```diff
@@ -61,13 +61,28 @@ if (!is_android) {
   }
 }
 
+action("cobalt_build_id") {
+  script = "build/build_id.py"
+  outputs = [ "$root_gen_dir/cobalt/cobalt_build_id.h" ]
+  args = [ rebase_path(outputs[0], root_build_dir) ]
+}
+
+# TODO(b/377927950): This should be consolidated with the above
+action("cobalt_build_info") {
+  script = "build/build_info.py"
+  outputs = [ "$root_gen_dir/build_info.json" ]
```

Comment:
Is this build_info.json output file actually used in this PR? It looks like `InitializeUserAgentPlatformInfoFields()` is just getting `COBALT_VERSION` from cobalt/version.h, which is an input to build_info.py.

I guess it must be used elsewhere (Kokoro?), or by something that will be added later?

Reply:
It's used downstream in Copybara->Kokoro->Piper flow. I'll consolidate the scripts+build rule and remove the `.txt` version of legacy build info as a follow-up, but for now just bringing the build rules back.